### DISCUSSION TITLE
Export to parent folder

### DISCUSSION THREAD

**Discussion by trecoool** [RandomEngy/VidCoder#1323] [Q&A] [answered]
Hello

I'm trying to get this structure automatically created by vidcoder when throwing the raw files at it :

```
Event/
   ├ raw/
   │   ├ file1.mov 
   │   └ file2.mov 
   ├ h265/
   │   ├ file1 h265.mp4
   │   └ file2 h265.mp4
   ├ file1 h264.mp4
   └ file2 h264.mp4
```

I have not been able to make it work with "../" or the {parent} variable in the custom naming format parameter of the picker.
Is it possible and did I miss something?

Thank you!

**Comment by RandomEngy** [comment, marked as answer]
Yeah, {parent} is just the name of the parent folder itself, so that wouldn't help you.

Checking "Output to source folder if possible" and using `..\{source}` as the custom file naming format worked for me.

**Comment by trecoool** [comment]
It works indeed.. not sure what i got wrong the first time 🤔
Thank you for your help
